src/main/java/kitchenpos/eatinorders/tobe/domain/order/EatInOrderLineItemQuantity.java

```diff
@@ -0,0 +1,39 @@
+package kitchenpos.eatinorders.tobe.domain.order;
+
+import java.util.Objects;
+
+public class EatInOrderLineItemQuantity {
```

Comment:
```suggestion

@Embeddable
public class EatInOrderLineItemQuantity {
```

해당 어노테이션이 있으면 더욱 좋지 않을까요?